test/e2e-test-application/src/luigi-config/extended/settings.js

```diff
@@ -56,8 +56,10 @@ class Settings {
   //     }
   //   };
   // };
+  // globalSideNavigation = true;
   // experimental = {
-  //   profileMenuFiori3: true
+  //    profileMenuFiori3: true,
+  //    globalNav: true
```

Comment:
If you want you can also add this to the experimental object. In that case all experimental features are listed.. (but of course not neccessary)
```suggestion
  //    globalNav: true,
  //    webcomponents:true
  
```

Reply:
added, sure